src/drivers/sdl/lv_sdl_window.c

```diff
@@ -338,7 +340,7 @@ static void window_create(lv_display_t * disp)
 #endif
 #endif /*LV_USE_DRAW_SDL == 0*/
     /*Some platforms (e.g. Emscripten) seem to require setting the size again */
-    SDL_SetWindowSize(dsc->window, hor_res * dsc->zoom, ver_res * dsc->zoom);
+    SDL_SetWindowSize(dsc->window, (int)((float)hor_res * dsc->zoom), (int)((float)ver_res * dsc->zoom));
```

Comment:
`hor_res` has already been multiplied by `dsc->zoom` earlier in this function. This sets the resolution to `width * zoom * zoom`. But zoom is always `1.0` because the display was just created.

So there are no problems, but for future maintenance maybe remove one of the zoom multiplies or remove all the zoom multiplies in this function because it will always be `1.0` in this function.

Reply:
Agreed. I have removed the scaling here.